dashboard/app/models/odometer.rb

```diff
@@ -0,0 +1,6 @@
+class Odometer < Widget
+
+  before_validation do
+    self.href = 'odometer/odometer.html'
```

Comment:
Yeah, just doing the following didn't work for me:
```ruby
def href
  'odometer/odometer.html'
end
```
Also this way it ends up in the `.level` file which makes it explicit when moving between environments (doesn't depend on any defaults existing in the model).